packages/liveblocks-node/src/PermissionSet.ts

```diff
@@ -0,0 +1,146 @@
+// As defined in the source of truth in ApiScope in
+// https://github.com/liveblocks/liveblocks-cloudflare/blob/main/src/security.ts
+const ALL_PERMISSIONS = Object.freeze([
+  "room:write",
+  "room:read",
+  "room:presence:write",
+  "comments:write",
+  "comments:read",
+] as const);
+
+export type Permission = (typeof ALL_PERMISSIONS)[number];
+
+function isPermission(value: string): value is Permission {
+  return (ALL_PERMISSIONS as readonly unknown[]).includes(value);
+}
+
+const MAX_PERMS_PER_SET = 10;
+
+/**
+ * Assign this to a room or room pattern if you want to grant the user
+ * read-only permissions for this room.
+ */
+export const READ_ACCESS: readonly Permission[] = Object.freeze([
+  "room:read",
+  // "room:presence:write",  // XXX Should this be included in the READ_ACCESS set?
+  "comments:read",
+]);
+
+/**
+ * Assign this to a room or room pattern if you want to grant the user all
+ * permissions for this room.
+ */
+export const FULL_ACCESS: readonly Permission[] = Object.freeze([
+  "room:write",
+  "comments:write",
+
+  // XXX Are these implied automatically if you have the two write permissions above? If so, it would be nice to keep the JWT payload as small as possible.
+  // "room:read",
+  // "room:presence:write",
+  // "comments:read",
```

Comment:
@ofoucherot I have a similar question here.

Reply:
Write implies read yes, you don't need to put both scopes.